src/main/java/org/radarcns/management/service/dto/SourceDTO.java

```diff
@@ -16,6 +16,9 @@
     @NotNull
```

Comment:
As we agreed a UUID should be assigned to the `sourceId` automatically by the system upon creation. This should be either handled either in `Domain` object using hibernate or in DTO's setter. Are we missing this?

Reply:
Indeed, it missed it in cherry-picking the commits from my working branch to this branch, it will come shortly in the next PR!